### ISSUE TITLE
Support for sm_86

### ISSUE THREAD

**Issue by arikhalperin** [facebookresearch/svoice#60]
Hello,
In order to support sm_86 we need to install pytorch1.90, but it breaks svoice. Is there a plan to upgrade?

Thanks,
Arik

**Comment by arikhalperin** [comment]
I managed to work with sm_86, with minor changes to code. Thinking about creating a PR for this, these are the requirements I had to change:
torch==1.7.1+cu110
torchaudio==0.7.2
torchvision==0.8.2+cu110
numpy==1.21.5

In code I had to do this(changed the call to the loss function):

        sisnr_loss, snr, est_src, reorder_est_src = cal_loss(
                        sources, estimate_source[c_idx], lengths)

I hope it helps

**Comment by gashishoo7** [comment]
Thanks for sm86 support. I tried using mentioned changes but it is giving error in torchaudio.load in cross validation part after completing one epoch training.
